Dockerfile.tpu

```diff
@@ -1,5 +1,4 @@
-ARG NIGHTLY_DATE="20250124"
-ARG BASE_IMAGE="us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.10_tpuvm_$NIGHTLY_DATE"
+ARG BASE_IMAGE="us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.6.0_3.10_tpuvm"
```

Comment:
do we have a way to uplevel this pin to a more recent nightly?

Reply:
On main the requirements are a torch 2.6.0 nightly (which no longer exists) and a torch_xla nightly which I believe was compiled against a torch 2.7.0 nightly.

I tried a few things that didn't work so the current iteration of the PR I tried to use stable versions of everything. In general it doesn't seem very sustainable to build vLLM against nightly versions of several different dependencies.

If that is how it must be for the TPU support, I'm happy to try a newer combination of nightlies.